Projects/B-U585I-IOT02A/tz_disabled/Src/mqtt/shadow_device_task.c

```diff
@@ -419,8 +422,8 @@ static bool prvSubscribeToShadowUpdateTopics( ShadowDeviceCtx_t * pxCtx )
     if(  xStatus == MQTTSuccess &&
          xSubSuccess == false )
     {
-        LogError( "Failed to register an incoming publish callback for topic %s.",
-                  pxCtx->pcTopicUpdateDelta );
+        LogError( ("Failed to register an incoming publish callback for topic %s.",
```

Comment:
Did you run into any build issues with the single-paren logging lines? The preprocessor should be able to handle either way (at least on gcc).

Reply:
Yes they are needed. Nothing will be printed on the serial console without them. Plus an error on CubeID
![LOG_LEVEL_Error](https://user-images.githubusercontent.com/41168224/156051596-eb8f55e5-9c32-4ec5-a799-f190771695b6.png)
E